svthalia/concrexit
# Provide a default value for existing partners

### Summary
In #1420 the country field was added to the model. However, no default field was provided and the field could be null. This is incorrect because the template will render the None value.

### How to test
Steps to test the changes you made:
1. Go to a partner page that existed before the original migration: https://staging.thalia.nu/career/partners/sapiente-delectus/
2. Check that in this new version there is a value and None is not shown.
